NetKVM/Common/ParaNdis_VirtQueue.cpp

```diff
@@ -163,7 +163,8 @@ bool CTXVirtQueue::Create(UINT Index,
     m_Context = Context;
 
     m_SGTableCapacity = m_Context->bUseIndirect ? virtio_get_indirect_page_capacity() : GetRingSize();
-
+    m_SGTableCapacity = min(m_SGTableCapacity, m_Context->uMaxFragmentsInOneNB);
```

Comment:
Its a little awkward to store this value twice back to back, but I understand for readability its not so bad